portal-wire-protocol.md

```diff
@@ -0,0 +1,158 @@
+# Portal Wire Protocol
+
+The Portal wire protocol is the base p2p protocol by which nodes communicate.
+
+The different Portal networks **MAY** use this base protocol, but they **MUST** remain separated per network.
+This is done at the [Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#talkreq-request-0x05) layer, by providing a different protocol bytestring, per network, in the `TALKREQ` message.
+
+The protocol bytestring in the `TALKREQ` message **MUST** have "portal-" as prefix then followed by the network name (e.g. "portal-state").
```

Comment:
Since this counts towards our byte allocation in the UTP packet, anyone have a strong opinion about using `prtl-` as our prefix and saving 2 bytes?  We can drop the dash `-` and save another byte.  These byte strings do not have to be human readable.  We could even go as far as simply defining them as a 3 byte prefix and a 1-byte network identifier.

Reply:
Adjusted this now to be somewhat shorter, but a bit silly as it becomes less and less readable, and thus we might as well go just with a 2 byte identifier for each network that we define here.

Unless the long term idea would be to really start these networks outside of discv5, and even then... (I do have some ideas regarding this but this is far from in scope for now ;) )